zrcs/zrc-7.md

```diff
@@ -0,0 +1,238 @@
+| `external_url`    | `object`         | A JSON object representing the external URL of the token.             | false    |
+| `value`           | `string`         | The URI pointing to an external website presenting the asset.         | false    |
+
+### External Description
+
+A textual description of the external resource for presenting the token. This is an optional field.
+
+| Name                     | Type             | Description                                                                   | Required |
+|--------------------------| ---------------- | ----------------------------------------------------------------------------- | -------- |
+| `external_description`   | `object`         | A JSON object representing the description of the external URL.               | false    |
+| `value`                  | `string`         | The description of the external url.                                          | false    |
+
+### Attributes
+
+An array of attributes with predefined types. Attributes can have the type of string, integer, decimal or date.
+
+| Name                     | Type             | Description                                                                   | Required |
+|--------------------------| ---------------- | ----------------------------------------------------------------------------- | -------- |
+| `attributes`             | `array`          | A JSON array of attributes with customised types of key value pairs.                                        | false    |
+
+| Name                     | Type             | Description                                                                   | Required |
+|--------------------------| ---------------- | ----------------------------------------------------------------------------- | -------- |
+| `type`                   | `*`              | The customised type of the attribute. Can be string, integer, decimal or date.           | false    |
+| `value`                  | `*`              | The value of the attribute.                                                   | false    |
+
+### Localization
+
+The data required for the sub-object is predefined, this should not be changed. This is a optional field.
+If the metadata contains a ```localization``` attribute, its content may be used to provide localized values for fields that need it.
+The localization attribute should be a sub-object with three attributes: uri, default and locales
+
+| Name            | Type             | Description                                                                    | Required |
+|-----------------| ---------------- | ------------------------------------------------------------------------------ | -------- |
+| `localization`  | `object`         | A JSON object representing the localization metadata.                          | false    |  |
+| `value`         | `string`         | A JSON object representing the localization properties.                        | false    |
```

Comment:
You are describing the schema from eip-1155 is represented in JSON.

Please understand the Metadata structure from https://docs.opensea.io/docs/metadata-standards#metadata-structure. Once you get it, you can simply provide a table as OpenSea does.